src/test/resources/features/99d_employment_qa_smoke.feature

```diff
@@ -0,0 +1,153 @@
+@mvp @ZAP @employment
+
+Feature: Smoke downstream Employment Journeys to DES in QA - Income Tax Submission
+
+  Background:
+    Given the user navigates to the auth login page
+
+  Scenario: Returning flow - Individual User with prior Employment, Review & change - Dividends Smoke
+    When the user logs into the service with the following details
+      |Redirect url       |/2022/start                                          |
```

Comment:
when i log in as this user in QA they dont have any employment data, is this local user data? cause if so it wont have data setup in qa